pkgs/applications/networking/remote/tlclient/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv
+, fetchurl
+, autoPatchelfHook
+, copyDesktopItems
+, makeDesktopItem
+
+, alsaLib
+, libX11
+, pcsclite
+}:
+let
+  tlclient-png = fetchurl {
+      url = "https://aur.archlinux.org/cgit/aur.git/plain/tlclient.png?h=tlclient&id=fd126edd54a012caa8c9ce47ab8d7fb25e982e74";
+      sha256 = "12i0wc4a8r69kwcdj0kjidwvmwhpb8jcwsqxw9dl5vm1gdqxyz5v";
+      name = "tlclient-png";
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "tlclient";
+  version = "4.12.0-6517";
+
+  src = fetchurl {
+    # x86_64 arch tarball
+    url = "https://www.cendio.com/downloads/clients/tl-${version}-client-linux-dynamic-x86_64.tar.gz";
+    sha256 = "0qpn28sh3p13f581gmdl3j59m40iwlpm6bpar7qiz1kr7cwlrzim";
+  };
+
+  nativeBuildInputs = [ autoPatchelfHook copyDesktopItems ];
+  propagatedBuildInputs = [ alsaLib libX11 pcsclite ];
+
+  dontBuild = true;
+
+  installPhase = ''
```

Comment:
FYI, it is recommended that you add both `preInstall` and `postInstall` hooks when overwriting phases: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md. This way, it avoids mistakes if someones needs to add a `preInstall` hook for example.

Reply:
But I am approving since for now it is looking right (the desktop file is included).

Reply:
Neat tool !